_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -3171,21 +3129,14 @@ public static function delete_service_api_key( $request ) {
 	 * @return string Returns the service name if valid, null if invalid.
 	 */
 	public static function validate_service_api_service( $service = null ) {
-		$valid_services = array(
-			'mapbox',
-		);
-		return in_array( $service, $valid_services, true ) ? $service : null;
+		return WPCOM_REST_API_V2_Endpoint_Service_API_Keys::validate_service_api_service( $service );
 	}
 
 	/**
 	 * Error response for invalid service API key requests with an invalid service.
 	 */
 	public static function service_api_invalid_service_response() {
-		return new WP_Error(
-			'invalid_service',
-			esc_html__( 'Invalid Service', 'jetpack' ),
-			array( 'status' => 404 )
-		);
+		return WPCOM_REST_API_V2_Endpoint_Service_API_Keys::service_api_invalid_service_response();
 	}
```

Comment:
I think we can omit this static method altogether since it's not directly used by this class anymore